swiss-ng-api/berlin-group-documentation/current.yaml

```diff
@@ -285,6 +286,8 @@ paths:
       responses: 
         '200':
           $ref: "#/components/responses/OK_200_PaymentInitiationInformation"
+        '202':
+          $ref: "#/components/responses/OK_200_PaymentInitiationInformation"
```

Comment:
Relevant Change: OK_202_PaymentInitiationStatus is new
Adoption approach: Follow the change

Reply:
removed again in 1.3.9. 2021-05-04v1 errata of 2021-06-30